Resources/Prototypes/_White/Actions/xenomorph.yml

```
@@ -0,0 +1,196 @@
+- type: entity
+  id: ActionTailLash
+  name: action-tail-lash
+  description: action-tail-lash-desc
+  categories: [ HideSpawnMenu ]
+  components:
+  - type: WorldTargetAction
+    useDelay: 11
+    range: 10
+    itemIconStyle: BigAction
+    checkCanAccess: false
+    icon: { sprite : _White/Interface/Actions/xenomorph.rsi, state: tail_attack_action }
```

YAMLlint (1.35.1)

[warning] too many spaces before colon

(colons)